packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -71,17 +71,15 @@ export function moduleFor(description, TestClass, ...mixins) {
     }
 
     if (name.indexOf('@test ') === 0) {
-      QUnit.test(name.slice(5), assert => context[name](assert));
+      QUnit.test(name.slice(5), assert => context[name](context.assert));
     } else if (name.indexOf('@skip ') === 0) {
-      QUnit.skip(name.slice(5), assert => context[name](assert));
+      QUnit.skip(name.slice(5), assert => context[name](context.assert));
     } else if (name.indexOf(packageTag) === 0) {
-      QUnit.test(name.slice(packageTag.length), assert => context[name](assert));
+      QUnit.test(name.slice(packageTag.length), assert => context[name](context.assert));
```

Comment:
Before, we were passing the global `QUnit.assert` here, and can certainly go back to that.